src/plugins/kibana_react/public/code_editor/code_editor.tsx

```diff
@@ -34,14 +34,14 @@ export interface Props {
   value: string;
 
   /** Function invoked when text in editor is changed */
-  onChange: (value: string) => void;
+  onChange: (value: string, event: monaco.editor.IModelContentChangedEvent) => void;
 
   /**
    * Options for the Monaco Code Editor
    * Documentation of options can be found here:
-   * https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.ieditorconstructionoptions.html
+   * https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.istandaloneeditorconstructionoptions.html
    */
-  options?: monaco.editor.IEditorConstructionOptions;
+  options?: monaco.editor.IStandaloneEditorConstructionOptions;
```

Comment:
We might be able to also grab the type from the react-monaco-editor [here](https://github.com/react-monaco-editor/react-monaco-editor/blob/master/src/types.ts#L18) in case this changes again the future but I think `IStandaloneEditorConstructionOptions` also works fine!